search/search-domain/src/main/java/io/camunda/search/page/SearchQueryPage.java

```diff
@@ -30,14 +33,23 @@ public Object[] startNextPageAfter() {
     return null;
   }
 
+  public SearchQueryPage sanitize() {
+    final var newFrom =
+        (from == null) ? DEFAULT_FROM : Math.max(0, Math.min(SearchQueryPage.DEFAULT_FROM, from));
```

Comment:
`newFrom` is always `0`, isn't?
if from >0; `min(0, from) = 0` and `max(0,0) =0`